Enterprise-CMCS/eAPD
# Alternatives and Risks

Resolves #4598 

### Description
Add the Analysis of Alternatives and Risks page. Fields are not required.

Also:
* cleaned up help text so that all example messages start with **Example:**, some were not bolded
* cleaned up some spelling errors
* cleaned up some instances where adminCheck wasn't being imported into some components or not being imported using the selector, also made adminCheck required in the propTypes of the components

### Chromatic Link
https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4614

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| an MMIS APD | the values are empty | nothing is displayed | jest |
| an MMIS APD | there are values | they are displayed | jest |
| an MMIS APD | values are entered | they are displayed | jest |

### Steps to manually verify this change

1. open an MMIS APD
2. navigated to the Activities Dashboard
3. open an activity
4. navigate to the Analysis of Alternatives and Risks page
5. verify that the values are empty
6. enter values in the fields
7. navigate away and back
8. verify that the values are still there
9. navigate to the state dashboard and back
10. verify that the values are still there

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
